src/firefly/js/tables/ui/ObjectTree.jsx

```diff
@@ -30,10 +32,9 @@ ObjectTree.propTypes = {
     defaultExpandAll:   PropTypes.bool,
     onSelect:           PropTypes.func,
     selectable:         PropTypes.bool,
-    className:          PropTypes.string,
     nodeClassName:      PropTypes.string,
-    style:              PropTypes.object,
-    title:              PropTypes.node
+    sx:                 PropTypes.object,
+    title:              PropTypes.oneOfType([PropTypes.node, PropTypes.string])
```

Comment:
Why specifically define `string` or `node` when node already includes string?

Reply:
Oh right. I think I confused the two because I'm explicitly converting string to Typography element

Reply:
I'll change node to element (I confused the two again 😄)